Formula/subversion.rb

```diff
@@ -23,23 +30,23 @@ class Subversion < Formula
   option "with-gnupg", "Build with support for GPG Agent"
 
   depends_on "pkg-config" => :build
+  depends_on "scons" => :build
   depends_on "swig" => :build
-  depends_on "apr-util"
   depends_on "apr"
+  depends_on "apr-util"
 
   # Always build against Homebrew versions instead of system versions for consistency.
   depends_on "lz4"
+  depends_on "openssl"
   depends_on "sqlite"
   depends_on "utf8proc"
   depends_on "perl" => :recommended
 
   # For Serf
```

Comment:
This comment needs to be moved with the deps.

Reply:
those are fixes by audit, maybe better to remove comments?